cookbook/product_asset/sftp.rst

```diff
@@ -0,0 +1,87 @@
+How to connect to an external server for storage
+================================================
+
+Quick Overview
+--------------
+
+**These cookbook examples are made for asset catalog storage but it works the same way for other catalogs**
+
+This cookbook assumes that you already configured an external server. This cookbook should be made before the installation of the pim. Else you will have to migrate your data to your external server.
+
+Connect to an sftp server
+-------------------------
+
+Lets assume you already configured an sftp server. You will have to setup the adapter in the app/config/config.yml file.
```

Comment:
Lets assume that you already have a fully functional sftp server ready to receive files.